src/main/java/edu/harvard/h2ms/service/report/ReportServiceImpl.java

```diff
@@ -39,14 +39,27 @@ public static ReportWorker getReportWorker(String reportType) {
 
   @Override
   public String requestReport(String reportType) {
-    log.info("report requested.  report type: " + reportType);
+    log.debug("report requested.  report type: " + reportType);
     log.debug("***** cache count" + reportWorkerCache.size());
 
     ReportWorker reportWorker = getReportWorker(reportType);
 
-    String ans = reportWorker.createReport();
+    String ans = reportWorker.createReport(null);

```

Comment:
Also, not important, but you could avoid writing a null check inside of createReport if you passed in an empty map instead of null.

Ex. `String ans = reportWorker.createReport(Collections.emptyMap());`